modules/core/test/test_io.cpp

```diff
@@ -664,7 +664,7 @@ static void test_filestorage_basic(int write_flags, const char* suffix_name, boo
                 f.close();
             }
             std::cout << "Storage size: " << sz << std::endl;
-            EXPECT_LE(sz, (size_t)6000);
+            EXPECT_LE(sz, (size_t)11000);
```

Comment:
This limit should not be changed.
This check passes on 3.4 branch and is aligned with BASE64 implementation.

Reply:
I checked the generated files manually and got it by myself. The flag is totally ignored in master branch: https://github.com/opencv/opencv/blob/master/modules/core/src/persistence.cpp#L452